client/ngapp/views/directives/ooni-info-country-list.directive.html

```diff
@@ -0,0 +1,15 @@
+
+<form class="filter-form">
+<div class="form-group inline">
+  <label>Filter:</label>
+  <input placeholder="eg. Iran" ng-model="countryFilter"/>
+</div>
+</form>
+
+<ul class="countries-list">
```

Comment:
@bnvk This directive is where the country list items are sketched out. Depending on what you'll be doing with each country item in the list, it might make sense to break each `li` into its own directive as well.